### PR TITLE
[usecase-wrt] Add test for handling SSL certificates

### PR DESCRIPTION
**Pull request by haoxli**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: Crosswalk for Android 21.51.546.0
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-1799

### CODE REVIEW

**Comment by haoxli** [comment]
@Honry

File: usecase/usecase-wrt-android-tests/samples/SSLCertificate/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Usecase Design
+
+This sample demonstrates the invalid SSL certificates and MITM request will be denied by Crosswalk, include:
+
+* https://crosswalk-project.org/jira/browse/XWALK-6986
+
+This usecase covers following ssl certificates issue:
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/SSLCertificate/README.md]: 's/certificates issue/certificate errors'

File: usecase/usecase-wrt-android-tests/samples/SSLCertificate/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+  <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <p><strong>Pre-condition:</strong></p>
+  <ol>
+    <li>Connect to available network.</li>
+    <li>Install mitmproxy in host device: https://mitmproxy.org/</li>
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/SSLCertificate/index.html]: Please use a accurate link to tell user how to install the mitmproxy.

File: usecase/usecase-wrt-android-tests/samples/SSLCertificate/index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+  <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <p><strong>Pre-condition:</strong></p>
+  <ol>
+    <li>Connect to available network.</li>
+    <li>Install mitmproxy in host device: pip install mitmproxy</br>
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/SSLCertificate/index.html]: Pls. add '$' before command line.

**Comment by Honry** [comment]
LGTM

**Pull request merged**